lib/Runtime/Language/JavascriptConversion.cpp

```diff
@@ -332,6 +339,11 @@ namespace Js
     //----------------------------------------------------------------------------
     Var JavascriptConversion::ToPrimitive(Var aValue, JavascriptHint hint, ScriptContext * requestContext)
     {
+        if (IsSimdType(aValue))
+        {
+            return aValue;
+        }
```

Comment:
Can you please move this under default? I am not why this should be the first case as other primitive types are more common